### DISCUSSION TITLE
Why is Firestore listed as availabe Auth Persistence Layer (APL) but docs are missing?

### DISCUSSION THREAD

**Discussion by madmonkeyworks** [saleor/apps#1078] [Q&A] [answered]
Hi there, 

In the Saleor docs on [https://docs.saleor.io/docs/3.x/developer/extending/apps/architecture/apl](https://docs.saleor.io/docs/3.x/developer/extending/apps/architecture/apl), there is **Firestore** listed as available Auth Persistence Layer, but I couldn't fine any related docs. 

Can anybody help? 
Thank you

**Comment by witoszekdev** [comment, marked as answer]
Hi @madmonkeyworks, sorry for the confusion, the Firestore APL was removed in favor of other APLs and our docs didn't reflect that.

Here's the current list of available APLs: https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-apps/app-sdk/apl#available-apls

_Thanks to your feedback, I've also updated the docs page you've linked_

If you'd like to develop your own APL that stores authorization data in Firestore you can check out these docs: https://docs.saleor.io/docs/3.x/developer/extending/apps/developing-apps/app-sdk/apl#example-implementation
